### PR TITLE
Add track class size feature

### LINKED ISSUE #117: Add track class size feature

### PR DESCRIPTION
**Pull request by adriantan101**
Fixes #117

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ title: AcademyAssist User Guide
 
 3. [Features](#features)
    * [Adding a student](#adding-a-student--add)
-   * [Deleting a student](#deleting-a-student--del)
+   * [Deleting a student](#deleting-a-student--delete)
```
Comment (JoanneHing) [inline docs/UserGuide.md:15]: Thanks for fixing the anchor bug for this link!

File: docs/_sass/minima/_base.scss [hunk 2]
```diff
@@ -288,7 +288,7 @@ table {
     text-align: center;
   }
   .site-header:before {
-    content: "AB-3";
+    content: "Academy Assist";
```
Comment (JoanneHing) [inline docs/_sass/minima/_base.scss:291]: And also thanks for fixing the header problem for this too!

File: src/main/java/seedu/academyassist/ui/HelpWindow.java [hunk 3]
```diff
@@ -35,12 +35,16 @@ public class HelpWindow extends UiPart<Stage> {
             + "   Format: delete STUDENT_ID\n\n"
             + "7. sort: arranges the list of students based on a specified field\n"
             + "   Format: sort s/FIELD\n\n"
-            + "8. clear: Clears all entries from the management system\n"
+            + "8. tracksubject: Displays how many students are taking each subject\n"
+            + "   Format: tracksubject\n\n"
+            + "9. filter: Filters contacts based on specified field\n"
+            + "   Format: filter f/FIELD [field]\n\n"
+            + "10. clear: Clears all entries from the management system\n"
```
Comment (JoanneHing) [inline src/main/java/seedu/academyassist/ui/HelpWindow.java:42]: Good that you remember to update the help window!

File: src/main/java/seedu/academyassist/ui/TrackSubjectWindow.java [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        this(new Stage(), model);
+    }
+
+    /**
+     * Updates the subject statistics display with current data from the model.
+     * Counts the number of students taking each subject and displays these counts
+     * in labeled format.
+     */
+    private void updateSubjectStats() {
+        Map<String, Integer> subjectCounts = new HashMap<>();
```
Comment (JoanneHing) [inline src/main/java/seedu/academyassist/ui/TrackSubjectWindow.java:57]: Good idea of using HashMap to store the `subjectCounts` with the Subject!

**Review by JoanneHing — COMMENTED** [review]
LGTM! Good job to notice those minor error of the current version!

File: docs/_sass/minima/_base.scss (hunk 2 above)
Comment (annabellekk) [reply docs/_sass/minima/_base.scss:291]: Nice catch!

**Review by annabellekk — APPROVED** [review]
LGTM

**Pull request merged**